backend/src/question/question.controller.ts

```diff
@@ -24,6 +31,16 @@ export class QuestionController {
       transform: true,
     })
   )
+  @ApiOperation({ summary: API_OPERATIONS.QUESTION_QUERY_SUMMARY })
+  @ApiOkResponse({
+    description: API_RESPONSES_DESCRIPTION.SUCCESSFUL_QUESTION_QUERY,
+  })
+  @ApiInternalServerErrorResponse({
+    description: API_RESPONSES_DESCRIPTION.INTERNAL_SERVER_ERROR,
+  })
+  @ApiNotFoundResponse({
+    description: API_RESPONSES_DESCRIPTION.NOT_FOUND_DESCRIPTION,
+  })
   @Get(["", "/summary"])
```

Comment:
actually, no use for "" route right? since it's internal usage (for developers)

Reply:
Yep, you're right. Now that I thought about it, doesnt make sense to have 2 endpoints that serves the same purpose -- will update to remove it.